src/compiler/php_plugin.cc

```diff
@@ -40,12 +41,25 @@ class PHPGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
       return true;
     }
 
+    std::vector<std::pair<grpc::string, grpc::string> > options;
+    ParseGeneratorParameter(parameter, &options);
+
+    grpc::string class_suffix;
+    for (size_t i = 0; i < options.size(); ++i) {
+      if (options[i].first == "class_suffix") {
```

Comment:
My understanding is that this is the preferred way (by way of how `protoc` works) of passing command line parameters to a protoc plugin. The `parameter` parameter is already there in the `Generate` method in the plugin already supplied by the main `protoc`. So we did not introduce that parameter per se. It's already passed to the plugin if you do specify something before the `:` in `--grpc_out`. This is just to formalize that, if you do pass something before the `:`, it should be in the format of `param1:var1,param2:var2`